src/services/positionService.ts

```diff
@@ -742,6 +834,35 @@ export default class PositionService {
     return this.safeService.submitMultipleTxs([approveTx, depositTx]);
   }
 
+  async xCallDeposit(
+    from: Token,
+    to: Token,
+    fromValue: string,
+    frequencyType: BigNumber,
+    frequencyValue: string,
+    yieldFrom?: string,
+    yieldTo?: string,
+    isPolygonDestnantion?: boolean
```

Comment:
there is a typo on Destination :)

Reply:
Is polygon the only destination that's going to be available?

Reply:
Same as above, sorry for typo lol